### DISCUSSION TITLE
PPOCRLabel标注使用中文输入法异常

### DISCUSSION THREAD

**Discussion by 77bw** [PaddlePaddle/PaddleOCR#13165] [Q&A]
#### 问题描述 / Problem Description
PPOCRLabel标注使用中文输入法异常 

https://github.com/PaddlePaddle/PaddleOCR/assets/94338396/4fd3d750-dae4-4699-8d45-3c5b12df90d7

#### 运行环境 / Runtime Environment
python 3.8
paddlepaddle==2.5.0 
PPOCRLabel==2.1.3

直接从官方文档的环境下载也不行

**Comment by GreatV** [comment]
换成英文输入法吧

**Reply by 77bw** [reply]
我是中文的，用英文不行呀

**Comment by UserWangZz** [comment]
ctrl+e对label进行编辑，可以使用中文输入法

**Reply by 77bw** [reply]
但是这样编辑文字太麻烦了

**Comment by GreatV** [comment]
https://github.com/PFCCLab/PPOCRLabel 在这提个issue吧
